tests/pragmas/thintprocess.nim

```diff
@@ -0,0 +1,16 @@
+discard """
+  matrix: "--hint:processing"
+  nimout: '''
+..
+warn_module.nim(6, 6) Hint: 'test' is declared but not used [XDeclaredButNotUsed]
+compile end
+'''
+"""
+
```

Comment:
rename the file to thintprocessing ? more searchable

Reply:
fixed